jx-args.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Problem: Potential security issue

- `jx-args.json` is used for dynamic, runtime configuration of workflows
- It can possibly contain secrets

Suggestion:

- We should put add it to `.gitignore`
- We should not bake it into Docker images at build time
- We should instead bind it into the container at runtime (or if there isn't a `jx-args.json` use the empty example file)

Created issue: https://github.com/AgPipeline/issues-and-projects/issues/234

Reply:
Note: This issue does not have to block this PR.